src/main/java/com/autotune/analyzer/utils/AnalyzerConstants.java

```diff
@@ -220,6 +224,494 @@ public int getSeverity() {
+        private NotificationCodes() {
+        }
+
+        // Section - Info:                  100000 - 199999
+        //      SubSection - General Info:  110000 - 119999
+        //          SubSystem - General:    110000 - 112999 (30% of availability)
+        //          SubSystem - CPU:        113000 - 113999 (10% of availability)
+        //          SubSystem - Memory      114000 - 114999 (10% of availability)
+        //          SubSystem - Network     115000 - 115999 (10% of availability)
+        //          SubSystem - Disk        116000 - 116999 (10% of availability)
+        //          SubSystem - Power       117000 - 117999 (10% of availability)
+        //      SubSection - Data Availability: 120000 - 129999
+        //          SubSystem - General:    120000 - 122999 (30% of availability)
+        //          SubSystem - CPU:        123000 - 123999 (10% of availability)
+        //          SubSystem - Memory      124000 - 124999 (10% of availability)
+        //          SubSystem - Network     125000 - 125999 (10% of availability)
+        //          SubSystem - Disk        126000 - 126999 (10% of availability)
+        //          SubSystem - Power       127000 - 127999 (10% of availability)
+
+        public static final int INFO_GENERAL_INFO_GENERAL_START = 110000;
+        // Subsystem section: Recommendation Engines
+        // Range: 112000 - 112999 (Each subsection can be given 100 entries which fit total of 10 entries or engines)
+
+        // Subsystem subsection: Default Engine
+        // Range: 112000 - 112099
+        // Subsystem subsection: Duration Based Engine
+        // Range: 112100 - 112199
+        public static final int DURATION_BASED_ENGINE_START = 112100;
+        public static final int DURATION_BASED_RECOMMENDATIONS_AVAILABLE = 112101;
+        public static final int DURATION_BASED_ENGINE_END = 112199;
+        // Subsystem subsection: Profile Based Engine
+        // Range: 112200 - 112299
+
+        public static final int INFO_GENERAL_INFO_GENERAL_END = 112999;
+        public static final int INFO_GENERAL_CPU_START = 113000;
+        public static final int INFO_GENERAL_CPU_END = 113999;
+        public static final int INFO_GENERAL_MEMORY_START = 114000;
+        public static final int INFO_GENERAL_MEMORY_END = 114999;
+        public static final int INFO_GENERAL_NETWORK_START = 115000;
+        public static final int INFO_GENERAL_NETWORK_END = 115999;
+        public static final int INFO_GENERAL_DISK_START = 116000;
+        public static final int INFO_GENERAL_DISK_END = 116999;
+        public static final int INFO_GENERAL_POWER_START = 117000;
+        public static final int INFO_GENERAL_POWER_END = 117999;
+        public static final int INFO_GENERAL_FUTURE_1_START = 118000;
+        public static final int INFO_GENERAL_FUTURE_1_END = 118999;
+        public static final int INFO_GENERAL_FUTURE_2_START = 119000;
+        public static final int INFO_GENERAL_FUTURE_2_END = 119999;
```

Comment:
Dont need these anymore, instead just use `RESERVED`

Reply:
Moved the constants to a better var names which describe their sections and subsections and subsystems as well